web_root/js/sockets.js

```diff
@@ -12,7 +12,7 @@
         var container = $("#content");
         var domain = window.location.host;
 
-        var socket = new WebSocket("ws://" + domain + "/ws");
+        var socket = new WebSocket("ws://" + domain + ":8080/ws");
```

Comment:
The :8080 part should already be included in location.host. I only tested on Chrome, what browser are you using?